Fluid/Ast/CommentStatement.cs

```diff
@@ -7,17 +7,12 @@ namespace Fluid.Ast
 {
     public class CommentStatement : Statement
     {
-        public CommentStatement(StringSegment text)
-        {
-            Text = text.ToString();
-        }
-
-        public CommentStatement(string text)
+        public CommentStatement(TextSpan text)
```

Comment:
maybe as `in` parameter, if saved as field property could be `public ref readonly TextSpan =>ref  _text;`